api/serverpb/server.proto

```diff
@@ -138,6 +140,9 @@ message ChangeSettingsRequest {
   bool remove_alert_manager_url = 8;
   string alert_manager_rules = 9;
   bool remove_alert_manager_rules = 10;
+  // Enable/disable Security Threat Tool
```

Comment:
This comment is added to Swagger documentation only for this field (you can see in generated serverpb.json above and in UI), and there is not way to disable STT with it. So let's add a proper comment to both fields.